config/initializers/simple_form.rb

```diff
@@ -164,7 +164,7 @@
 
```

Comment:
I realise this was a comment from potentially a long time ago, but [html5 validations are very well supported](https://caniuse.com/#feat=form-validation). Seeing as this is Koi, which is even more strict about browser support than the front-end I'd say it's pretty safe. 
Even [Ornament now has it turned on by default](https://github.com/katalyst/ornament/blob/master/test/dummy/config/initializers/simple_form.rb#L116). 

Client-side validations are very dumb, all you can match is a regex-like pattern and requiredness. This is not in any way a replacement for rails validations. All I'm planning on using it for is setting requiredness on some fields. 

We may need to think about how to display errors if submitted data fails a backend check which I haven't considered at all. Since it's technically a JSON blob, I'm really not even sure where to begin! I'm happy to hear any ideas on this issue.